Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -1472,30 +1505,8 @@ def addCKFTracks(
         )
     )
     cutSets = [
-        acts.TrackSelector.Config(
-            **acts.examples.defaultKWArgs(
-                loc0Min=c.loc0[0],
-                loc0Max=c.loc0[1],
-                loc1Min=c.loc1[0],
-                loc1Max=c.loc1[1],
-                timeMin=c.time[0],
-                timeMax=c.time[1],
-                phiMin=c.phi[0],
-                phiMax=c.phi[1],
-                etaMin=c.eta[0],
-                etaMax=c.eta[1],
-                absEtaMin=c.absEta[0],
-                absEtaMax=c.absEta[1] if len(tslist) == 1 else None,
-                ptMin=c.pt[0],
-                ptMax=c.pt[1],
-                minMeasurements=c.nMeasurementsMin,
-                maxHoles=c.maxHoles,
-                maxOutliers=c.maxOutliers,
-                maxHolesAndOutliers=c.maxHolesAndOutliers,
-                maxSharedHits=c.maxSharedHits,
-                maxChi2=c.maxChi2,
-                measurementCounter=c.nMeasurementsGroupMin,
-            )
+        convertPyTrackSelectorConfig(
+            c, lambda etaMax: etaMax if len(tslist) == 1 else None
```

Comment:
alternatively we could convert he python named tuple to a kwargs so we can still overwrite `absEtaMax` later. the lambda looks a bit arbitrary and hacky

Reply:
yeah I wanted to do this with least possible changes, and also didn't spend a lot time into thinking about this... But I don't see a way to do this within the list comprehension actually... would you do another loop?